website/events/emails.py

```diff
@@ -46,15 +46,20 @@ def notify_organiser(event, registration):
     :param event: the event
     :param registration: the registration that was cancelled
     """
-    if event.organiser is None or event.organiser.contact_mailinglist is None:
+    if event.organisers.count() < 1 is None:
         return
 
-    text_template = get_template("events/organiser_email.txt")
-    subject = f"Registration for {event.title} cancelled by member"
-    text_message = text_template.render({"event": event, "registration": registration})
+    for organiser in event.organisers.all():
+        if organiser.contact_mailinglist is None:
+            return
+        text_template = get_template("events/organiser_email.txt")
+        subject = f"Registration for {event.title} cancelled by member"
+        text_message = text_template.render(
+            {"event": event, "registration": registration}
+        )
 
-    EmailMessage(
-        subject,
-        text_message,
-        to=[event.organiser.contact_mailinglist.name + "@thalia.nu"],
-    ).send()
+        EmailMessage(
+            subject,
+            text_message,
+            to=[organiser.contact_mailinglist.name + "@thalia.nu"],
```

Comment:
Instead of repeatedly sending an email you could also add them all to the 'to' line of the one email. Then they also see that they all got the notification.

Reply:
Or batch the messages together so that it requires only one smtp connection :)

Reply:
Lets do the first